k8s_audit_config/apiserver-config.patch.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+IFS=''
+
+FILENAME=${1:-/etc/kubernetes/manifests/kube-apiserver.yaml}
+VARIANT=${2:-minikube}
+
+if grep audit-webhook-config-file $FILENAME ; then
+    echo audit-webhook patch already applied
+    exit 0
+fi
+
+TMPFILE="/tmp/kube-apiserver.yaml.patched"
+rm -f "$TMPFILE"
+
+APISERVER_PREFIX="    -"
+APISERVER_LINE="- kube-apiserver"
+
+if [ $VARIANT == "kops" ]; then
+    APISERVER_PREFIX="     "
+    APISERVER_LINE="/usr/local/bin/kube-apiserver"
+fi
+
+while read LINE
+do
+    echo "$LINE" >> "$TMPFILE"
+    case "$LINE" in
+        *$APISERVER_LINE*)
+            echo "$APISERVER_PREFIX --audit-log-path=/var/lib/k8s_audit/audit.log" >> "$TMPFILE"
```

Comment:
What if the k8s apiserver has audit enabled with a different path? Are we going to break the existing configuration?

Reply:
The script would shortcut (see https://github.com/draios/sysdig-cloud-scripts/pull/57/files/ec6441790f94bb9a620628fc11d0ff5b211534ca#diff-3eead3fcf4dd36d03053bb66c1cecfd8R8). So it's safe to run. If somebody wants to add a *second* audit log destination, then I think they should make their own changes without using these instructions/scripts directly.

Reply:
Sorry, I didn't mean the `audit-webhook-config-file` check, but rather the `audit-log-path`, let say k8s audit enabled with audit log write to a PV `/some-folder/audit.log`, I think the the script will break the existing configuration. The script serves well in general purpose, but the field engineers need to pay attention to the details.

Reply:
You're right. I'll add a note to the README that this assumes no audit config of any kind is in place.